Blockzilla/SettingsViewController.swift

```diff
@@ -416,7 +458,9 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
                 cell.accessibilityIdentifier = "settingsViewController.trackingCell"
                 cell.accessoryType = .disclosureIndicator
             } else {
-                cell = getToggleCell(indexPath: indexPath)
+                let toggle = toggleForIndexPath(indexPath)
+                cell = SettingsTableViewToggleCell(style: .subtitle, reuseIdentifier: "toggleCell", toggle: toggle)
```

Comment:
This is a lot of code duplication right now (these three lines are repeated thrice) are we able to reduce it somehow?